integrations/langchain/src/databricks_langchain/chat_models.py

```diff
@@ -62,41 +62,7 @@
 class ChatDatabricks(BaseChatModel):
     """Databricks chat model integration.
 
-    Setup:
-        Install ``databricks-langchain``.
-
-        .. code-block:: bash
-
-            pip install -U databricks-langchain
-
-        If you are outside Databricks, set the Databricks workspace hostname and personal access token to environment variables:
-
-        .. code-block:: bash
-
-            export DATABRICKS_HOSTNAME="https://your-databricks-workspace"
-            export DATABRICKS_TOKEN="your-personal-access-token"
-
-    Key init args — completion params:
```

Comment:
Should we keep these descriptions?

Reply:
Its not needed. Using pydantic docs, it would automatically pick it up from args mentioned here: https://github.com/databricks/databricks-ai-bridge/pull/60/files#diff-d3df2a94788772d7305c1d27a61e633d133fc5dcb66fec30ce9b918136f67e44L247
Right now its duplicated so its confusing